### PR TITLE
Refactor/calendar/utils css

### PR DESCRIPTION
**Pull request by S-chahine**
Details:

- Moved inline css styles from DayEvent, MonthEvent, and WeekEvent files into seperate css files.
- Combined the css styles for both DayEvent and MonthEvent into DayMonthEventStyles.css
- Moved Weekevent styles into seperate WeekEventStyles.css
- Moved isMultiDay and formatEventTime functions into dataUtils.jsx

### CODE REVIEW

**Review by ElenaCherpakova — COMMENTED** [review]
Where do we use this component `DayViewHeader` in `DayEvent.jsx` file or is it just a helper function?

**Comment by ElenaCherpakova** [comment]
There is some small remarks: 
We name our CSS files just .css, which can lead to class name conflicts. To prevent this, please name your CSS files with the .module.css extension  like so `DayMonthEventStyle.module.css ` .  This will enable CSS modules, generating unique class names and preventing class name collisions.

File: frontend/src/components/Calendar/DayMonthEventStyles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.titleStyle {
```
Comment (ElenaCherpakova) [inline frontend/src/components/Calendar/DayMonthEventStyles.css:1]: could you please add module.css  to style file like so DayMonthEventStyles.module.css?

**Review by ElenaCherpakova — CHANGES_REQUESTED** [review]

File: frontend/src/components/Calendar/DayMonthEventStyles.css (hunk 1 above)
Comment (S-chahine) [reply frontend/src/components/Calendar/DayMonthEventStyles.css:1]: I initially thought we only needed to change the names of the CSS files. However, it turns out that we also need to update the className assignments in the code. Should I proceed with making these changes now?

**Comment by S-chahine** [comment]
I found out that the DayViewHeader function was originally placed in the DayEvent component, even though it is only used in KFCalendar. To improve the organization and maintainability of our code, I have moved the DayViewHeader function from DayEvent to the dateUtils file and imported it into KFCalendar.

File: frontend/src/components/Calendar/DayMonthEventStyles.css (hunk 1 above)
Comment (ElenaCherpakova) [reply frontend/src/components/Calendar/DayMonthEventStyles.css:1]: It wont require too much changes you just need to add module it and export it like so: 
import styles from './Button.module.css'; 
and add that styles before style name 
[Adding a CSS Modules Stylesheet](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet)

Comment (S-chahine) [reply frontend/src/components/Calendar/DayMonthEventStyles.css:1]: I've been working on modifying the CSS file names and updating the class names in the code. Unfortunately, the styles aren't coming through as intended.
I think the problem is that we're using React Big Calendar, and we're trying to override its default styles to match our Figma design. This seems to be causing some issues.
Do you have any ideas on how we can fix this while keeping our design consistent?

**Review by ElenaCherpakova — APPROVED** [review]

**Pull request merged**